integration_tests/service_c/src/schema.py

```diff
@@ -1,22 +1,26 @@
 from graphene import ObjectType, String, Int, List, NonNull, Field
-from graphene_federation import build_schema, extend, external, requires, key
+from graphene_federation import build_schema, extend, external, requires, key, provides
 
 
 @extend(fields='id')
 class User(ObjectType):
     id = external(Int(required=True))
     primary_email = external(String())
     uppercase_email = requires(String(), fields='primaryEmail')
+    age = external(Int())
 
     def resolve_uppercase_email(self, info):
         return self.primary_email.upper() if self.primary_email else self.primary_email
 
+    def resolve_age(self, info):
+        return 18
+
 
 @key(fields='id')
```

Comment:
`@key` decorator was added in another PR
Thus test passed
we need to use type without `@key` decorator

Reply:
resolved